### PR TITLE
LPS-132496 Asset is related to DLFileEntry not FileEntry

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

File: modules/apps/info/info-list-provider-item-selector-web/src/main/java/com/liferay/info/list/provider/item/selector/web/internal/layout/list/retriever/InfoItemRelatedListProviderLayoutListRetriever.java [hunk 1]
```diff
@@ -128,6 +126,16 @@ private InfoListProviderContext _getInfoListProviderContext() {
 		return new DefaultInfoListProviderContext(group, user);
 	}
 
+	private String _getModelClassName(Object contextObject) {
+		if (contextObject instanceof FileEntry) {
+			return FileEntry.class.getName();
```
Comment (victorg1991) [inline modules/apps/info/info-list-provider-item-selector-web/src/main/java/com/liferay/info/list/provider/item/selector/web/internal/layout/list/retriever/InfoItemRelatedListProviderLayoutListRetriever.java:140]: I guess we can iterate over all the parent classes/implementation and check if we have something in the info framework for that.. not sure if it is worth it. We only have this special case

**Comment by jkappler** [comment]
looks good @victorg1991!

**Comment by jkappler** [comment]
ci:forward

**Pull request closed without merging**